nsv13/code/modules/overmap/overmap.dm

```diff
@@ -445,6 +445,8 @@ Proc to spool up a new Z-level for a player ship and assign it a treadmill.
 	if(ai_controlled)
 		weapon_types[FIRE_MODE_MISSILE] = new/datum/ship_weapon/missile_launcher(src)
 		weapon_types[FIRE_MODE_TORPEDO] = new/datum/ship_weapon/torpedo_launcher(src)
+	if(broadside)
```

Comment:
It might be a good idea to handle these via overrides on specific ships (like how the whisp/shrike handle specific guns) instead of adding a single var like this.

Reply:
We should honestly refactor the entire system to not require these dumb datums. Now any new type of gun built on the ship won't be able to fire

Reply:
probably a bit out of scope for this though

Reply:
The entire weapon handling code is a hot mess sadly. Yeah, it does need rewriting if you have the expertise.

A simple thing to point out is that we literally have zero interface between what has been loaded versus what is being fired, ie there is no actual way of pathing to select ammo type from a pool etc.